bundlewrap/items/pkg_freebsd.py

```diff
@@ -0,0 +1,90 @@
+from shlex import quote
+
+from bundlewrap.exceptions import BundleError
+from bundlewrap.items import Item
+from bundlewrap.utils.text import mark_for_translation as _
+
+def parse_pkg_name(pkgname, line):
+    # Contains the assumption that version may not contain '-', which is covered
+    # according to the FreeBSD docs (Section 5.2.4, "PKGNAMEPREFIX and PKGNAMESUFFIX")
+    installed_package, _, installed_version = line.rpartition('-')
```

Comment:
```suggestion
    installed_package, _sep, installed_version = line.rpartition('-')
```

You are redefining _ here, so it's no longer an alias for mark_for_translation (gettext function), but a string returned from str.rpartition().

Reply:
Oh, that's it! Thanks, I would never have found that :see_no_evil: